kevm-pyk/src/kevm_pyk/cli.py

```diff
@@ -37,7 +38,7 @@ class KEVMCLIArgs(KCLIArgs):
     @cached_property
     def target_args(self) -> ArgumentParser:
         args = ArgumentParser(add_help=False)
-        args.add_argument('--target', choices=['llvm', 'haskell', 'haskell-standalone', 'foundry'])
+        args.add_argument('--target', type=KompileTarget, default=KompileTarget.LLVM, choices=list(KompileTarget))
```

Comment:
This is now used not for kompiling (`kevm kompile ...` which is now `kevm kompile-spec ...`), but just for `kevm kast ...` and `kevm run ...`, for selecting which prebuilt backend to use for executing those commands. In that case, it doesn't make sense to use `__TARGET__`, because doing `kevm kast --target plugin ...` wouldn't make sense.

Reply:
`KompileTarget` values correspond to templates for kompilation (with values `haskell` and `llvm`). Here, we want to refer to one of the pre-built definitions (with values `haskell`, `haskell-standalone` and `llvm`.

Reply:
Done.